raster/r.resamp.filter/r.resamp.filter.html

```diff
@@ -37,8 +43,33 @@ <h2>DESCRIPTION</h2>
 
 <p>
 All of the kernels specified by the <b>filter</b> parameter are
-multiplied together. Typical usage will use either a single kernel or an
-infinite kernel along with a finite window.
+multiplied together. Typical usage will use either a single finitie
+window or an infinite kernel along with a finite window.
+
+<h3>Usage hints</h3>
+<p>
+To smooth a map, keeping its resolution, a good starting point is to
+use the filters <b>gauss,box</b> with the radii <em>0.5 * input
+resolution</em>, <em>2 * input resolution</em>. See also <a
+href="r.neighbors.html">r.neighbors</a>
+
+<p>
+When resampling a map to a higher resolution (alternative to
+interpolation, e.g. <a
+href="r.resamp.interp.html">r.resamp.interp</a>), a good starting point
+is to use the filters <b>gauss,box</b> with the radii <em>1.5 * input
+resolution</em>, <em>3 * input resolution</em>.
+
+<p>
+When resampling a map to a lower resolution (alternative to
+aggregation, e.g. <a href="r.resamp.stats.html">r.resamp.stats</a>), a
+good starting point is to use the filters <b>gauss,box</b> with the
+radii <em>0.25 * output resolution</em>, <em>1 * output
+resolution</em>.
+
+<p>
+These are recommendations for initial settings, filters and radii might
+need adjustment according to the actual purpose.
```

Comment:
I've reread a couple of times to understand what was meant, the comma was kind of weird

Reply:
Better now?

Reply:
Yep!